desktop/src/app/localizer/desktop-localizer.ts

```diff
@@ -0,0 +1,51 @@
+import { Localizer } from "@batch/ui-common/lib/localization";
+import * as path from 'path';
+import { ipcRenderer } from "electron";
+import * as fs from 'fs';
+import { AppLocaleService } from "app/services";
+
+interface Translations {
+    [key: string]: string;
+}
+
+export class DesktopLocalizer implements Localizer {
+    private translations?: Translations;
+
+    constructor(private localeService: AppLocaleService) { }
+
+    async loadTranslations(): Promise<void> {
```

Comment:
Instead of implementing a loadTranslations() function here, can we just read the map on AppTranslationsLoaderService, which should already have all of the localized strings?